public/app/plugins/datasource/grafana-azure-monitor-datasource/dashboards/v1Alerts.json

```
@@ -93,7 +71,7 @@
                   {
                     "targetBlank": false,
                     "title": "",
-                    "url": "d/dyzn5SK7z/alert-consumption?${sub:queryparam}﻿﻿﻿﻿﻿&﻿﻿﻿﻿﻿${rg:queryparam}﻿﻿﻿﻿﻿&﻿﻿﻿﻿﻿${__url_time_range}﻿﻿﻿﻿﻿&var-mc=Fired&var-mc=Resolved&var-as=New&var-as=Acknowledged&var-as=Closed&var-sev=Sev0&var-sev=Sev1&var-sev=Sev2&var-sev=Sev3&var-sev=Sev4&${__url_time_range}"
+                    "url": "d/dyzn5SK7z/alert-consumption?${sub:queryparam}&${rg:queryparam}&${__url_time_range}&var-mc=Fired&var-mc=Resolved&var-as=New&var-as=Acknowledged&var-as=Closed&var-sev=Sev0&var-sev=Sev1&var-sev=Sev2&var-sev=Sev3&var-sev=Sev4&${__url_time_range}"
```

these URLs are still wrong, as far as I can see, you need to replace in all of them:

```suggestion
                    "url": "d/dyzn5SK7z/azure-alert-consumption?${sub:queryparam}&${rg:queryparam}&${__url_time_range}&var-mc=Fired&var-mc=Resolved&var-as=New&var-as=Acknowledged&var-as=Closed&var-sev=Sev0&var-sev=Sev1&var-sev=Sev2&var-sev=Sev3&var-sev=Sev4&${__url_time_range}"
```

Please, make sure the code works for you

---

Thanks for the catch, I also had to add the data source variable in the URL!